README.md

```
@@ -38,20 +34,20 @@ A robust authentication microservice built with [Deno](https://deno.land/) and [
 
 ## 📋 Table of Contents
 
-- [Prerequisites](#prerequisites)
-- [Setup](#setup)
-- [Environment Variables](#environment-variables)
-- [API Endpoints](#api-endpoints)
-- [Usage](#usage)
-- [Technologies Used](#technologies-used)
-- [License](#license)
+- [Prerequisites](#-prerequisites)
+- [Setup](#-setup)
```

Markdownlint (0.37.0)

Link fragments should be valid

(MD051, link-fragments)